### DISCUSSION TITLE
remix grunge stack .. aws

### DISCUSSION THREAD

**Discussion by riadraz** [remix-run/remix#8429] [Q&A] [answered]
After deploying in aws , i got mail from github that says run failed grunge stack.. some jobs can no be successful. .. i also got deployed link. Clicking the url shows  {"message":"Internal Server Error"}. I am very new to aws lambda , any help will appreciated. I saw someone youtube stuffs on grunge stack and they abled to run the the stack clicking the the url aws provided.

**Comment by gustavorino** [comment, marked as answer]
First, check if the cloudformation stack was deployed successfully. You can check that on aws.

Secondly, that internal server error is either coming from API gateway or Lambda. Check the logs on cloudwatch. It might be a permission or a missing env variable.

Sorry to reply with something vague, but I could be more helpful if you provide details.

**Reply by riadraz** [reply]
Thank you very much.. Yes it was several attempts of 5-7 times and at the last i could deploy it. You are correct of seesion env variable mismatch was the deploy failure error. My cloud watch log also showed lot of logs ,,, among them that one also existed. I am new here and want to explore more on aws and remix combination.
